plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/graph/transform/ElkGraphImporter.java

```diff
@@ -479,7 +479,8 @@ private LGraph createLGraph(final ElkNode elkgraph) {
         // since size information stored there may apply to the current graph node)
         ElkPadding nodeLabelpadding = NodeLabelAndSizeCalculator.computeInsideNodeLabelPadding(
                 elkgraph.getParent() == null ? null : ElkGraphAdapters.adapt(elkgraph.getParent()),
-                ElkGraphAdapters.adaptSingleNode(elkgraph));
+                ElkGraphAdapters.adaptSingleNode(elkgraph),
```

Comment:
Formatting looks off :).

Reply:
Oh, how so? I tried to put each argument on a separate line.

Reply:
Usually you would still increase the indentation after the first line break. At least that's what feels natural to me.

Reply:
Oh, interesting, I've never done it that way. 😀

Reply:
Actually, I misread the diff. You added a parameter to the `computeInsideNodeLabelPadding` method, not the `adaptSingleNode`. Hence the indentation I was referring to happens in line 481.